### PR TITLE
Server and database

### PR DESCRIPTION
**Pull request by bayan-404**
connected next to the database using [pg-promise ](https://www.npmjs.com/package/pg-promise)
and still figuring out a way to add the database files
_I deleted the client  folder because we don't have a custom server_  (that's why 16 file is effected)

### CODE REVIEW

File: pages/api/getUsers.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import db from '../../lib/db';
+
+export default async (req, res) => {
+  try {
+    const users = await db.one('SELECT * FROM users;');
+    res.status(200).json(users);
+  } catch (err) {
+    res.json(err);
+    res.status(500).end();
```
Comment (Jood80) [inline pages/api/getUsers.js:9]: can't you type that like ` res.status(500).json(err).end()` ? or `res.status(500).send(err)`

File: pages/api/getUsers.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import db from '../../lib/db';
+
+export default async (req, res) => {
+  try {
+    const users = await db.one('SELECT * FROM users;');
```
Comment (YakoobHammouri) [inline pages/api/getUsers.js:5]: move SQL query to separate file

File: pages/api/getUsers.js (hunk 1 above)
Comment (YakoobHammouri) [inline pages/api/getUsers.js:9]: did you check this 
you can use like this 
return res.status(500).json(err)

File: pages/api/getUsers.js (hunk 2 above)
Comment (Jood80) [inline pages/api/getUsers.js:5]: it seems that the semicolon isn't mandatory here to execute your query,
http://vitaly-t.github.io/pg-promise/Database.html#one

**Review by Jood80 — APPROVED** [review]

File: pages/api/getUsers.js (hunk 2 above)
Comment (bayan-404) [reply pages/api/getUsers.js:5]: I know , I just like to follow the low 🚀

Comment (bayan-404) [reply pages/api/getUsers.js:5]: ☑️ done

File: pages/api/getUsers.js (hunk 1 above)
Comment (bayan-404) [reply pages/api/getUsers.js:9]: that's clean ✨ ty

File: package.json [hunk 3]
```diff
@@ -9,7 +9,9 @@
   },
   "dependencies": {
     "@material-ui/core": "latest",
+    "axios": "^0.21.1",
```
Comment (Rahaf-96) [inline package.json:12]: I don't think you need `axios` now

Comment (bayan-404) [reply package.json:12]: Yeah I was testing but we will need it later so I will keep it

Comment (Rahaf-96) [reply package.json:12]: cool

**Pull request merged**